### DISCUSSION TITLE
How i can get list of the supported fonts?

### DISCUSSION THREAD

**Discussion by RAprogramm** [kovidgoyal/kitty#7810] [Q&A] [answered]
before i got list, but now interactive changer....

**Comment by kovidgoyal** [comment]
Why would you want one?

**Comment by RAprogramm** [comment]
> Why would you want one?
> 

I had a plugin for neovim, to manage fonts directly from neovim, and quite a few people used it, but now the functionality is limited

https://github.com/NeViRAIDE/nekifoch.nvim

**Comment by kovidgoyal** [comment, marked as answer]
kitty +runpy 'from kitty.fonts.common import all_fonts_map; import json; print(json.dumps(all_fonts_map(True), indent=2))'

**Reply by RAprogramm** [reply]
> kitty +runpy 'from kitty.fonts.common import all_fonts_map; import json; print(json.dumps(all_fonts_map(True), indent=2))'

is this command return only installed font supported in kitty or all supported in kitty or all installed fonts?

**Comment by kovidgoyal** [comment]
And here is a simple function that shows how to extract the platform independent data from the output of all_fonts_map.

https://github.com/kovidgoyal/kitty/commit/ccc3bee9af794f332b4e9adcd714a649f639c397

**Comment by kovidgoyal** [comment]
it returns the same set of fonts as by the old list-fonts

**Reply by RAprogramm** [reply]
thank you very much!
